VersionControlUI/src/DiffManager.cpp

```diff
@@ -174,6 +174,26 @@ QSet<Model::Node*> DiffManager::findAllNodesWithDirectParentPresent(QHash<Model:
 	return nodesMarkedForRemoval;
 }
 
+QSet<Visualization::Item*> DiffManager::findAllItemsWithParentPresent(QSet<Visualization::Item*> items)
+{
+	QSet<Visualization::Item*> itemsMarkedForRemoval;
```

Comment:
The name of this local field is out of place. Nothing in this method itself suggests what the resulting list will be used. You're just computing which are the items that have parents, and somebody on the outside uses this to remove some items from somewhere.

Call  this `result`